packages/graphql/src/schema/create-relationship-fields/index.ts

```diff
@@ -668,11 +680,19 @@ function createRelationshipFields({
                 count_GTE: "Int",
                 AND: `[${relationshipWhereTypeInputName}!]`,
                 OR: `[${relationshipWhereTypeInputName}!]`,
+                NOT: relationshipWhereTypeInputName,
                 ...(nodeWhereAggregationInput ? { node: nodeWhereAggregationInput } : {}),
                 ...(edgeWhereAggregationInput ? { edge: edgeWhereAggregationInput } : {}),
             },
         });
 
+        const deprecates_NOT = {
+            name: "deprecated",
+            args: {
+                reason: "Negate filters will be deprecated from version 4.0.0, use the NOT operator to achieve the same behavior",
+            },
+        };
```

Comment:
I see this is declared in multiple places. Perhaps we can introduce a new file to import it from?